### PR TITLE
Added blog link

### PR DESCRIPTION
**Pull request by lillietricianijiiro**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a live link or preview screenshot
- [ ] Images are `240 x 240` [w x h]
- [ ] Resolved merge conflicts
- [ ] Included a description of my change below

# Things done in this Pull Request

- Replace this with your change description

### CODE REVIEW

**Comment by akshatnitd** [comment]
@lillietricianijiiro Please resolve the conflicts :)

**Review by anshumanv — CHANGES_REQUESTED** [review]
Please upload your image.

**Review by the-codingninja — CHANGES_REQUESTED** [review]
Upload your image and resolve conflicts

**Review by isuruAb — COMMENTED** [review]
Please upload your image and resolve confllicts

**Review by abhinavk96 — CHANGES_REQUESTED** [review]
The actual png image file needs to be uploaded.

File: _data/blogs.yml [hunk 1]
```diff
@@ -130,5 +130,5 @@
   
 - url: https://patriciascodeintime.wordpress.com
   title: My journey with Google Code-in
-  img: Part 1 of blog.jpg, Part 2 of blog.jpg, Part 3 of blog.jpg, Part 4 of blog.jpg, Part 5 of blog.jpg
+  img: Part 1 of blog.jpg, Part 2 of blog.jpg, Part 3 of blog.jpg, Part 4 of blog.jpg, Part 5 of blog.jpg
```
Comment (abishekvashok) [inline _data/blogs.yml]: Please provide a definite img name, like blogname_wihout_spaces_short.png

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by abishekvashok** [comment]
There are also ocnflicts pls resolve them :)

**Comment by AmanSingh200** [comment]
https://wordpress.com/read/feeds/77017858/posts/1711640617

can someone plzz add this blog

**Comment by ParthS007** [comment]
@lillietricianijiiro  Closing this PR , Please read `README.md` and create a new PR 👍

**Pull request closed without merging**